tests/ga/test_policy_engine.py

```diff
@@ -28,8 +27,22 @@ class TestPolicyEngine(AgentTestCase):
     patcher = None
     regorus_dest_path = None    # Location where real regorus executable should be.
     default_policy_path = os.path.join(data_dir, 'policy', "agent-extension-default-data.json")
-    default_rule_path = os.path.join(data_dir, 'policy', "agent_extension_policy.rego")
-    input_json = None  # Input is stored in a file, and extracted into this variable during class setup.
+    default_rule_path = os.path.join(data_dir, 'policy', "agent_policy.rego")
+    input_json = {
+        "extensions": {
+            "Microsoft.Azure.ActiveDirectory.AADSSHLoginForLinux": {
+                "signingInfo": {
+                    "extensionSigned": False
+                }
+            },
+            "test2": {
```

Comment:
Could we rename this to TestExtension2, same for test3 below

Reply:
Done